project/nsw-state-library-amplify/project.json

```diff
@@ -9,7 +9,8 @@
   "allowTranscriptDownload": true,
   "gaCode": "",
   "adminEmails": [
-    "glen.crawford@reinteractive.net"
```

Comment:
yeah i was wondering about this, if i'm the only user, have they not been using the current admin panel at all?